common/components/Image/Image.tsx

```diff
@@ -32,7 +44,14 @@ export const Image: FC<ImageProps> = ({
       loading="lazy"
       className={styles.img}
       src={src}
-      alt=""
+      onError={e => {
+        (e.target as HTMLImageElement).onerror = null;
+        (e.target as HTMLImageElement).srcset =
+          fallbackImageSrcSets
+            ?.map(image => `${image.src} ${image.width}w`)
+            .join(",") || "";
+      }}
+      alt="fallback"
```

Comment:
Image is decorative, so no need for alt text. If we wanted to use an alt text for the fallback image it should give more context to the user like "image not visible". But since we don't use alt text for visible images, I don't think we need it for fallback either.

Reply:
Fikset